README.md

```diff
@@ -312,20 +316,63 @@ A few additional links to some typical early `Jupyter` topics:
 * [Closing running Jupyter notebook servers](https://github.com/jupyter/notebook/issues/2844)
 * [Checkpoints and autosave](https://groups.google.com/forum/#!topic/jupyter/DGCKE5fS4kQ)
 
-### Continuous Integration
+#### Command-line Interface (CLI)
+
+Python's ecosystem offers several ways to tackle command-line interfaces. The traditional standard method is to use
+the `argparse` module that is part of the standard library. This can be complemented by something like `argparsetree`
+for larger and more complex command-line applications.
+
+Here we have chosen to use [Click](https://click.palletsprojects.com/) instead, which allows us to define our CLI via
+decorated functions in a neat and compact way. Other potential alternatives could
+be [docopt](https://docopt.readthedocs.io/) or [Invoke](https://www.pyinvoke.org/).
+
+A nice comparison is
+available [here](https://realpython.com/comparing-python-command-line-parsing-libraries-argparse-docopt-click/).
+
+In order to install the package and **run the CLI commands**, you can follow the steps below.
```

Comment:
```suggestion
In order to install the package and **run the CLI commands**, you can follow these steps:
```

Reply:
I wrote it the way I did because it's not just a single concise bullet list that follows but visually interrupted